msos/WaitChains.cs

```diff
@@ -56,8 +56,7 @@ public void Execute(CommandExecutionContext context)
 
         private void SetStrategy()
         {
-            if (_context.TargetType == TargetType.DumpFile
-                || _context.TargetType == TargetType.DumpFile)
```

Comment:
I am quite sure this was supposed to be `DumpFileWithoutHeap` or whatever the constant is called -- I'm on mobile so I can't check right now.